Customizing/global/plugins/Services/Repository/RepositoryObject/TalentAssessment/classes/Observations/class.ilMyObservationsGUI.php

```
@@ -56,10 +56,31 @@ public function executeCommand() {
 			case "performAllAssFilter":
 				$this->mode = self::MODE_ALL;
 				$this->performFilter();
+			case "showPDF":
+				$this->showPDF();
 				break;
 		}
 	}
 
+	protected function showPDF() {
+		$this->mode = $_GET["mode"];
+
+		require_once($this->plugin->getDirectory()."/classes/class.ilObjTalentAssessment.php");
+		$ta_obj = new ilObjTalentAssessment($_GET["xtas_ref_id"]);
+		$settings = $ta_obj->getSettings();
+		$actions = $ta_obj->getActions();
+
+		$pdf = new TalentAssessment\Observations\ilResultPDF($settings, $actions, $this->txt);
+		$file_name = "TA_".$settings->getFirstname()."_".$settings->getLastname();
+		try {
+			$pdf->show($file_name, "D");
+		} catch(\Exception $e) {
```

Empty Catch-All is really bad. It's basically the error silencer (@).